### ISSUE TITLE
netlify

### ISSUE THREAD

**Issue by oeyoews** [izuolan/notionic#12]
不支持netlify吗，页面总是找不到，vercel一切正常的
![1E4179BF-0BC3-4BE1-9B4E-24C82CCE76F1](https://user-images.githubusercontent.com/72405338/207773175-38fc9389-ff48-4743-8400-61a9347a1358.png)
![CF3B5D33-5C06-401D-AD04-3DCE256EC0EC](https://user-images.githubusercontent.com/72405338/207773179-cdc36790-facb-45d9-90d0-c96176b18731.png)

**Comment by izuolan** [comment]
确实, 我在本地 next build 和 netlify dev 都是正常的, 但是 netlify build deploy 之后就找不到页面了, 我也不太清楚什么问题.
(>_<)
